src/Cryptol/REPL/Command.hs

```diff
@@ -1929,6 +1949,12 @@ findNbCommand :: Bool -> String -> [CommandDescr]
 findNbCommand True  str = lookupTrieExact str nbCommands
 findNbCommand False str = lookupTrie      str nbCommands
 
+-- | Look up a command, then return its help message as a helpful error message.
+findCmdHelp :: String -> String
+findCmdHelp cmd = unlines ["ERROR", "Command Description: " ++ cHelp descmd]
+  where 
+    [descmd] = findCommandExact cmd
```

Comment:
This is likely to lead to warnings, so it might be better to rewrite like a case:
```
descmd =
  case findCommandExact cm dof
    [cmd] -> cmd
    _ ->panic "findCmdHelp" [ "Unexpected result of `findCommandExact`", cmd ]
``` 
Although if we change the error message above, you probably don't need to find the command.
Also, we may want to rename the function to something like `invalidCommandArgument`, as it wouldn't be showing command help anymore.